src/lib/openapi/spec/project-stats-schema.ts

```diff
@@ -18,30 +18,57 @@ export const projectStatsSchema = {
     properties: {
```

Comment:
This schema doesn't have a description, so it's hard to understand what it's for. This is also a good place to explain the "last 30 days" and "previous 30 days before current window" things.

But what is the previous window? Is it 60-31 days ago? 61-31 days ago? 60-30 days ago? What does "the last 30 days" mean? Is today day 1 or day 0?

```suggestion
    description: "Statistics for a project, including the average time to production, number of features created, the project activity and more.
    
Stats are divided into current and previous **windows**. 
- The **current window** is the past 30 days. 
- The **previous window** is the 30 days **before** the current window (from 60 to 30 days ago)",
    properties: {
```